PyInstaller/hooks/hook-PyQt5.py

```diff
@@ -18,16 +19,36 @@
                                    'PyQt5')
          if os.path.basename(x[0]) == 'qt.conf']
 
-# Include ICU files, if they exist. See the "Deployment approach" section in
-# ``PyInstaller/utils/hooks/qt.py``.
-[(os.path.join(pyqt5_library_info.location['BinariesPath'], dll),
-  os.path.join('PyQt5', 'Qt', 'bin', dll))
- for dll in ('icudt??.dll', 'icuin??.dll', 'icuuc??.dll')]
-
-# TODO: Include software rendering for OpenGL. See the "Deployment approach". However, because the standard PyQt5 wheel `doesn't include <https://www.riverbankcomputing.com/pipermail/pyqt/2018-June/040387.html>`_ ``d3dcompiler_XX.dll``, this produces failures. When the wheel is updated, this code can be uncommented.
-##binaries = []
-##for dll in ('libEGL.dll', 'libGLESv2.dll', 'd3dcompiler_??.dll', 'opengl32sw.dll'):
-##    dll_path = os.path.join(pyqt5_library_info.location['BinariesPath'], dll)
-##    # Only add files if they exist.
-##    if glob(dll_path):
-##        binaries += [(dll_path, os.path.join('PyQt5', 'Qt', 'bin', dll))]
+
+def include_all_or_none(globs_to_include):
+    """
+    globs_to_include is a list of file name globs
+    Each glob can only match one file.
+    Every file must be available in order for any in the provided list to be
+    included.
+    """
+    to_include = []
+    for dll in globs_to_include:
+        dll_path = os.path.join(pyqt5_library_info.location['BinariesPath'],
+                                dll)
+        dll_file_paths = glob.glob(dll_path)[:1]
```

Comment:
Would you add a comment here, explaining the `[:1]`? "Allow exactly one match" or something like that?